src/components/screens/monitor/delegates/index.js

```diff
@@ -13,32 +14,70 @@ import withLocalSort from '../../../../utils/withLocalSort';
 import withResizeValues from '../../../../utils/withResizeValues';
 import NotAvailable from '../notAvailable';
 
-const defaultUrlSearchParams = { tab: 'active' };
+const defaultUrlSearchParams = { search: '' };
 const delegatesKey = 'delegates';
+const standByDelegatesKey = 'standByDelegates';
 
 const mapStateToProps = ({ blocks: { latestBlocks } }) => ({
   latestBlocks,
 });
 
+const transformResponse = (response, oldData, urlSearchParams) => (
+  urlSearchParams.offset
+    ? [...oldData, ...response.filter(
+      delegate => !oldData.find(({ username }) => username === delegate.username),
+    )]
+    : response
+);
+
 const ComposedDelegates = compose(
   withRouter,
-  withData({
-    [delegatesKey]: {
-      apiUtil: liskService.getDelegates,
-      defaultData: [],
-      defaultUrlSearchParams,
-      autoload: true,
-      transformResponse: (response, oldData, urlSearchParams) => (
-        urlSearchParams.offset
-          ? [...oldData, ...response.filter(
-            delegate => !oldData.find(({ username }) => username === delegate.username),
-          )]
-          : response
-      ),
+  withData(
+    {
+      [delegatesKey]: {
+        apiUtil: liskService.getActiveDelegates,
+        defaultData: [],
+        autoload: true,
+        transformResponse,
+      },
+
+      [standByDelegatesKey]: {
+        apiUtil: liskService.getStandbyDelegates,
+        defaultData: [],
+        autoload: true,
+        transformResponse,
+      },
+
+      chartActiveAndStandbyData: {
+        apiUtil: liskService.getActiveAndStandByDelegates,
+        defaultData: [],
+        autoload: true,
+        transformResponse: response => response.total,
+      },
+
+      chartRegisteredDelegatesData: {
+        apiUtil: liskService.getRegisteredDelegates,
+        defaultData: [],
+        autoload: true,
+        transformResponse: (response) => {
+          const responseFormatted = response.reduce((acc, delegate) => {
+            const newDelegate = { ...delegate, timestamp: moment(delegate.timestamp * 1000).startOf('month').toISOString() };
```

Comment:
This part is ambigeous. please explain what are you trying to achive in comments.

Reply:
I fixed